meta-resin-common/recipes-devtools/btrfs-tools/btrfs-tools_%.bbappend

```diff
@@ -1,2 +1,9 @@
 FILESEXTRAPATHS_append := ":${THISDIR}/${PN}"
-SRC_URI_append = " file://0001-btrfs-progs-utils-make-sure-set_label_mounted-uses-c.patch"
+
+# for btrfs-tools versions older than 4.5.1 we need to still apply the following patch:
+python() {
+    packageVersion = d.getVar('PV', True)
+    srcURI = d.getVar('SRC_URI', True)
+    if packageVersion < '4.5.1':
```

Comment:
That's a good point. Maybe we should start using something like https://pypi.python.org/pypi/semver if the package's versions are semver compliant. Otherwise the problem gets a little more complicated.